database/pom.xml

```
@@ -136,24 +132,29 @@
         <database.driver>com.ibm.db2.jcc.DB2Driver</database.driver>
         <database.datasource.class>com.ibm.db2.jcc.DB2SimpleDataSource</database.datasource.class>
       </properties>
-
-      <dependencies>
-        <dependency>
-          <groupId>com.ibm.db2.jcc</groupId>
-          <artifactId>db2jcc4</artifactId>
-          <scope>test</scope>
+    </profile>
+    <profile>
+      <id>db2-115</id>
+      <properties>
+        <version.db2>${version.db2-11.5}</version.db2>
+      </properties>
+	  <dependencies>
+	    <dependency>
+          <groupId>com.ibm.db2</groupId>
+          <artifactId>jcc</artifactId>
+		  <version>${version.db2}</version>
         </dependency>
-      </dependencies>
-      <build>
+	  </dependencies>
+	  <build>
         <pluginManagement>
           <plugins>
             <plugin>
               <groupId>org.codehaus.mojo</groupId>
               <artifactId>sql-maven-plugin</artifactId>
               <dependencies>
-                <dependency>
-                  <groupId>com.ibm.db2.jcc</groupId>
-                  <artifactId>db2jcc4</artifactId>
+	            <dependency>
```

```suggestion
                <dependency>
```

The formatting looks a bit off.